docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,187 @@
+        - determine the UEF based on the draw pattern for MEDIUM: `if draw_pattern == "MEDIUM": expected_uef = 0.9307 – (0.0002 × storage_volume)`
+        - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.9349 – (0.0001 × storage_volume)`
+        - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+          - check if the actual uniform_energy_factor equals the expected: `if uniform_energy_factor == expected_UEF:`
+            - set rule_status to "PASS": `rule_status = "PASS"; CONTINUE TO RULE ASSERTION`
+          - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note += "  The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") does not equal required (" + expected_UEF + ")"; CONTINUE TO RULE ASSERTION`
+        - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note += "  Uniform Energy Factor is required to evaluate water heaters in ASHRAE 90.1 Table 7.8 / 10 CFR 430.  Uniform Energy Factor is not provided for this waterheater.  Note that the specific requirements of 10 CFR 430 can be found in ASHRAE 90.1 Appendix F Table F-2."; CONTINUE TO RULE ASSERTION`
+
+      - else, if the storage tank is greater than 55 and less than 100 gallons, Appendix F provides a heat pump efficiency, which may be an error.  return UNDETERMINED: `elif storage_volume <=100 and storage_volume > 55:`
+        - set rule_status to UNDETERMINED: `rule_status = "UNDETERMINED"; CONTINUE TO RULE ASSERTION`
+        - include a note: `rule_note += "  The storage tank is between 55 and 100 gallons with a capacity <= 12kW (40945 btu/hr). The minimum efficiency requirements in 90.1 Section 7.4.2 point to 10 CFR 430 which provides a heat pump efficiency for water heaters in this capacity range which is not consistent with the efficiency of an electric resistance storage water heater which is the only electric SWH system type associated with the baseline for ASHRAE 90.1 Appendix G. Consequently, this rule was not able to be assessed for this service water heater.  Note that the specific requirements of 10 CFR 430 can be found in ASHRAE 90.1 Appendix F Table F-2."`
+        - go to rule assertion: `GO TO RULE ASSERTION`
+ 
+    - otherwise, the capacity per volume is not  < 4000 btu/ gallon: `else:`
+      - indicate with the note that the capacity per volume exceeds the limit: `rule_note += "  Capacity per volume exceeds the limit of 4000 btu/hr/gallon given for Electric Storage Water Heaters in ASHRAE 90.1 Table 7.8."`
+      - set the rule status to FAIL and continue to rule assertion: `rule_status = "FAIL"; CONTINUE TO RULE ASSERTION`
+
+    - else if the storage volume is between 2 and 20 gallons, in which case there is no rating in APPENDIX F.  set rule status to UNDETERMINED: `elif storage_volume < 20:`
```

Comment:
Bit confused by this logic - should this be nested inside the capacity per volume?

Reply:
you are correct - it should've been inside the capacity per volume check